### ISSUE TITLE
"parameter name Subnets does not exist"

### ISSUE THREAD

**Issue by arokem** [richford/cloudknot#193]
@36000 is currently seeing the following error: 

```
---------------------------------------------------------------------------
WaiterError                               Traceback (most recent call last)
<ipython-input-9-5ca797b2ee75> in <module>
      5                   resource_type="SPOT",
      6                   bid_percentage=100,
----> 7                   memory=64000)
~/cloudknot/cloudknot/cloudknot.py in __init__(self, name, pars, pars_policies, docker_image, base_image, func, image_script_path, image_work_dir, image_github_installs, username, repo_name, image_tags, job_definition_name, job_def_vcpus, memory, retries, compute_environment_name, instance_types, resource_type, min_vcpus, max_vcpus, desired_vcpus, image_id, ec2_key_pair, bid_percentage, job_queue_name, priority)
   1439             }
   1440
-> 1441             self._pars, pars_cleanup = futures["pars"].result()
   1442
   1443             self._docker_image, self._docker_repo, repo_cleanup = futures[
~/miniconda3/envs/afq/lib/python3.7/concurrent/futures/_base.py in result(self, timeout)
    433                 raise CancelledError()
    434             elif self._state == FINISHED:
--> 435                 return self.__get_result()
    436             else:
    437                 raise TimeoutError()
~/miniconda3/envs/afq/lib/python3.7/concurrent/futures/_base.py in __get_result(self)
    382     def __get_result(self):
    383         if self._exception:
--> 384             raise self._exception
    385         else:
    386             return self._result
~/miniconda3/envs/afq/lib/python3.7/concurrent/futures/thread.py in run(self)
     55
     56         try:
---> 57             result = self.fn(*self.args, **self.kwargs)
     58         except BaseException as exc:
     59             self.future.set_exception(exc)
~/cloudknot/cloudknot/cloudknot.py in set_pars(knot_name, input_pars, pars_policies_)
   1321                 else:
   1322                     try:
-> 1323                         pars_ = Pars(name=knot_name, policies=pars_policies_)
   1324                     except aws.CannotCreateResourceException:
   1325                         pars_ = Pars(
~/cloudknot/cloudknot/cloudknot.py in __init__(self, name, batch_service_role_name, ecs_instance_role_name, spot_fleet_role_name, policies, use_default_vpc, ipv4_cidr, instance_tenancy)
    395                     "stack_create_complete"
    396                 )
--> 397                 waiter.wait(StackName=self._stack_id, WaiterConfig={"Delay": 10})
    398
    399                 response = aws.clients["cloudformation"].describe_stacks(
~/miniconda3/envs/afq/lib/python3.7/site-packages/botocore/waiter.py in wait(self, **kwargs)
     51     # method.
     52     def wait(self, **kwargs):
---> 53         Waiter.wait(self, **kwargs)
     54
     55     wait.__doc__ = WaiterDocstring(
~/miniconda3/envs/afq/lib/python3.7/site-packages/botocore/waiter.py in wait(self, **kwargs)
    321                     name=self.name,
    322                     reason='Waiter encountered a terminal failure state',
--> 323                     last_response=response,
    324                 )
    325             if num_attempts >= max_attempts:
WaiterError: Waiter StackCreateComplete failed: Waiter encountered a terminal failure state
```

In the CloudFormation console he's seeing an error that says:  "Parameter validation failed: parameter value for parameter name Subnets does not exist."

**Comment by richford** [comment]
@36000, could you share the python command that initiated this error? I'm guessing it was a `ck.knot` or `ck.pars` instantiation.

**Comment by richford** [comment]
Hmm. I'm not sure how to rephrase to please SentimentBot. @36000, no offense intended. I'm pretty sure the bot got it wrong here. But honestly, please do let me know if it's off-putting to you. I want to take seriously the possibility that I'm messing up here.

**Comment by arokem** [comment]
Maybe it's objecting to the fact that you didn't say "please"  🤣

**Comment by richford** [comment]
Maybe I can spell it like "Puh-leeze"

**Comment by richford** [comment]
![](https://media.giphy.com/media/gQDazMXigdtD2/giphy.gif)

**Comment by arokem** [comment]
This is resolved with #193 merged.
